src/config/models.py

```diff
@@ -82,6 +82,12 @@ def set_defaults(cls, data: "AffixFilterCountModel") -> "AffixFilterCountModel":
             data["maxCount"] = len(data["count"])
         return data
 
+    @field_validator("minCount")
+    def min_count_validator(cls, minCount: int) -> int:
```

Comment:
should we also do max count?

Reply:
max count should also be at least 1 I guess then, right?

Reply:
max count should be [1,5]. upper boundary is not as important as it simply wouldn't match. but we can catch this user error